Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/Settings.java

```diff
@@ -184,6 +236,11 @@ public void saveSettings(SettingsActivityView view, Context context)
 
       NativeConfig.save(NativeConfig.LAYER_LOCAL_GAME);
     }
+
+    for (Map.Entry<File, IniFile> entry : mWiimoteProfileFiles.entrySet())
+    {
+      entry.getValue().save(entry.getKey());
```

Comment:
This would become `entry.getValue().save(SettingsFile.getWiiProfile(entry.keyKey()))`.